news/_posts/2016-02-07-evb-ev3dev-lang-java-release-0.2.0.md

```diff
@@ -0,0 +1,170 @@
+---
+author: "@jabrena"
+title: "Announcing EV3Dev lang Java v0.2.0"
+---
+
+If you like Linux features from EV3Dev and you enjoy with Java, this library is the solution to develop Java software for robots on EV3Dev. [EV3Dev-lang-java](https://jabrena.github.io/ev3dev-lang-java/#/) is a Java library designed to interact with the hardware managed by EV3Dev using the LeJOS way.
+
+**What features are included in last release?**
+
+Current version includes this features:
+
+* Regulated Motor Support
+* Unregulated Motor Support
+* Sensor Support
+* Sounds
+* LeJOS Sensor filter Support
+* eSpeak TTS (Text to speech) Support
+
+JavaDocs: https://jabrena.github.io/ev3dev-lang-java/docs/api/
```

Comment:
This will not be a hyperlink. Try `JavaDocs: <https://jabrena.github.io/ev3dev-lang-java/docs/api/>` instead.